config.c

```diff
@@ -1504,6 +1504,11 @@ int git_default_config(const char *var, const char *value, void *cb)
 		return 0;
 	}
 
+	if (starts_with(var, "gvfs.")) {
```

Comment:
I think I want to move this somewhere else.  probably in a separate iteration so that we don't alter the main init sequence and so that we can cluster all of the fields we want to echo to trace2.